src/definition/metric-semaphores.js

```diff
@@ -0,0 +1,106 @@
+const metricSemaphores = {
+  type: 'items',
+  label: 'Metric Semaphores',
+  items: {
+    AllMetrics: {
+      ref: 'allmetrics',
+      type: 'boolean',
+      component: 'switch',
+      label: 'All metrics affected',
+      options: [
+        {
+          value: true,
+          label: 'On'
+        },
+        {
+          value: false,
+          label: 'Off'
+        }
+      ],
+      defaultValue: false
+    },
+    MetricsAffected: {
+      ref: 'metricssemaphore',
+      translation: 'Metrics affected (1,2,4,...)',
+      type: 'string',
+      defaultValue: '0',
+      show (data) {
+        return data.allmetrics == false;
+      }
+    },
+    MetricStatus1: {
+      ref: 'metricsstatus1',
+      translation: 'Critic is less than',
+      type: 'number',
+      defaultValue: -0.1
+    },
+    ColorStatus1: {
+      ref: 'colorstatus1',
+      label: 'Critic Color Fill',
+      type: 'object',
+      component: 'color-picker',
+      defaultValue: {
+        index: 7,
+        color: '#f93f17'
+      }
```

Comment:
dualOutput: true,
is missing to prevent (colors are off by 1 issue)

Reply:
Can't fix everything in one go, alot of these changes are just moving existing code around, this for example. Fixing the color issues will be left for the usual coloring issue